requirements/edx/testing.txt

```diff
@@ -105,7 +105,7 @@ edx-celeryutils==1.0.0    # via -r requirements/edx/base.txt, super-csv
 edx-completion==4.0.3     # via -r requirements/edx/base.txt
 edx-django-release-util==1.0.0  # via -r requirements/edx/base.txt
 edx-django-sites-extensions==3.0.0  # via -r requirements/edx/base.txt
-edx-django-utils==3.15.0  # via -r requirements/edx/base.txt, django-config-models, edx-drf-extensions, edx-enterprise, edx-rest-api-client, edx-toggles, edx-when, ora2, super-csv
+-e git+https://github.com/edx/edx-django-utils.git@master#egg=edx-django-utils  # TODO: Change this to a proper version.
```

Comment:
Once, edx/edx-django-utils#109 is merged, we will:
1. Revert this change.
1. Bump the required version from `edx-django-utils>=3.12.0` to `3.16.0` in `edx/base.in`.
1. Run `make upgrade` to pin the new version.